src/js/components/Search/SearchResults.tsx

```diff
@@ -1,84 +1,32 @@
 import React from 'react';
-import { Card, Col, Row, Statistic, Typography, Space } from 'antd';
-import { TeamOutlined } from '@ant-design/icons';
-import { BiDna } from 'react-icons/bi';
-
-import { PieChart } from 'bento-charts';
-import CustomEmpty from '../Util/CustomEmpty';
-import ExpSvg from '../Util/ExpSvg';
-
-import { CHART_HEIGHT, COUNTS_FILL } from '@/constants/overviewConstants';
-import { useAppSelector, useTranslationDefault } from '@/hooks';
+import { useAppSelector } from '@/hooks';
+import SearchResultsPane from './SearchResultsPane';
 
 const SearchResults = () => {
-  const t = useTranslationDefault();
-
   const isFetchingData = useAppSelector((state) => state.query.isFetchingData);
-
   const biosampleCount = useAppSelector((state) => state.query.biosampleCount);
   const biosampleChartData = useAppSelector((state) => state.query.biosampleChartData);
-
   const experimentCount = useAppSelector((state) => state.query.experimentCount);
   const experimentChartData = useAppSelector((state) => state.query.experimentChartData);
-
   const individualCount = useAppSelector((state) => state.query.individualCount);
   const message = useAppSelector((state) => state.query.message);
 
-  const wrapperStyle = {
-    padding: '40px',
-    minHeight: '150px',
-    maxHeight: '475px',
-  };
+  // existing search code has message existing as only sign of insufficent data
+  const hasInsufficientData = Boolean(message)
```

Comment:
equality might be clearer, Boolean() is not a common pattern in JS - also missing semi

Reply:
I'll just remove this variable entirely if you think implicit is preferable: ie use `hasInsufficientData={message}` in the call to the component

Reply:
It will have to be `message !== undefined` since the current expected value of "message" is either undefined, or the string "Insufficient data available." This is the nature of the katsu response, which could always be improved. 

I'll throw that in if you can live with it.

Reply:
I think that sounds good!

Reply:
Looking further into the redux it seems like "message" is given a default value of empty string if it's missing from the katsu response, so  `message !== ""` should be enough for now.